src/visual/VisualGroup.js

```diff
@@ -0,0 +1,58 @@
+import joint from 'jointjs';
+
+const cDefaultWidth = 100;
+const cMinHeight = 100;
+const cPixelPerSymbol = 15;
+const cHeightPerPort = 50;
+
+/** Class that provides graphical representation for the Step.
+ * @private
+ */
+export default class VisualGroup extends joint.shapes.devs.Model {
+  constructor(step, opts = {}) {
+    super({
+      position: {
+        x: (opts.x - cDefaultWidth / 2) || 0,
+        y: (opts.y - cMinHeight / 2) || 0,
+      },
+      attrs: {
+        '.label': {
+          text: step.name,
+        },
+      },
+      type: 'VisualStep',
+    });
```

Comment:
Please, check if it is OK to have two different Model classes with the same type name.